wordle-web/components/PossibleWords.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div>
+    <v-select
+      clearable
+      :label="labelText"
+      v-model="selectedWord"
+      :items="wordList"
+      variant="solo-filled"
+      :menu-props="{ bottom: true }"
+    ></v-select>
```

Comment:
```suggestion
    <v-select
      clearable
      :label="labelText"
      v-model="selectedWord"
      :items="wordList"
    />
```
Use self closing tag here. I removed `:menu-props="{ bottom: true }` prop from your v-select and it didn't seem to change anything so consider removing it

Reply:
I was trying out the menu-props option, but forgot to remove it after it didn't change anything thank you for pointing that out.